detection-rules/attachment_pdf_extortion_leveraging_breach_data.yml

```diff
@@ -0,0 +1,50 @@
+name: "Extortion / Sextortion - PDF attachment leveraging breach data from freemail sender"
+description: "Detects sextortion attempts leveraging breach data, including names, addresses, phone numbers and frequently using Google Maps/Bing Maps streetview images to bolster confidence and fear."
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+
+  // sender is a freemail domain
+  and sender.email.domain.root_domain in $free_email_providers
+
+  // attachment filename is the same as the subject
+  and any(attachments,
+          strings.ilevenshtein(strings.concat(subject.subject,".",
+                                .file_extension), .file_name) <= 1
+  )
+  
+  // body contains a US address, or the body is the subject
+  and (
+    regex.icontains(body.current_thread.text,
+                    '\d+\s[\w\s.]+(?:\n)?[\w\s]+\s[A-Z]{2}\s\d{5}(?:-\d{4})?(?:\n)?'
+    )
+    or subject.subject == body.current_thread.text
+  )
+  
+  // there's a PDF attachment with an image at a depth of one, measuring 148x148 and containing a QR code that is a BTC address
+  and any(attachments,
+          .file_type == "pdf"
+          and any(file.explode(.),
+                  (.depth == 1 and .flavors.mime == "image/jpeg")
+                  and .scan.exiftool.image_height == 148
+                  and .scan.exiftool.image_width == 148
```

Comment:
Do you feel this is too specific?

Reply:
I don't think so, it seems to be whatever they're generating the QR code with has a specific size.